### PR TITLE
chars: init at 0.5.0

### PR DESCRIPTION
**Pull request by bbigras**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/chars/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv
```
Comment (IvarWithoutBones) [inline pkgs/tools/text/chars/default.nix:1]: `stdenv` is never used, just the `lib` attribute for the meta section.
```suggestion
{ lib
```

File: pkgs/tools/text/chars/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  src = fetchFromGitHub {
+    owner = "antifuchs";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1pyda3b6svxzc98d7ggl7v9xd0xhilmpjrnajzh77zcwzq42s17l";
+  };
+
+  cargoSha256 = "1ampmw0l2wk2xp4q13aj5shxncqfh4dc3rsmpk2scaivanrsikn5";
+
+  meta = with stdenv.lib; {
```
Comment (IvarWithoutBones) [inline pkgs/tools/text/chars/default.nix:19]:
```suggestion
  meta = with lib; {
```

**Review by IvarWithoutBones — CHANGES_REQUESTED** [review]
1 nit, otherwise LGTM 👍

**Comment by bbigras** [comment]
uh. TIL. Thanks and done!

**Review by IvarWithoutBones — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
```patch
diff --git a/pkgs/tools/text/chars/default.nix b/pkgs/tools/text/chars/default.nix
index 18b29066ac56..fa58d110550e 100644
--- a/pkgs/tools/text/chars/default.nix
+++ b/pkgs/tools/text/chars/default.nix
@@ -1,6 +1,7 @@
-{ lib
+{ stdenv
 , fetchFromGitHub
 , rustPlatform
+, Security
 }:
 
 rustPlatform.buildRustPackage rec {
@@ -16,7 +17,9 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "1ampmw0l2wk2xp4q13aj5shxncqfh4dc3rsmpk2scaivanrsikn5";
 
-  meta = with lib; {
+  buildInputs = stdenv.lib.optionals stdenv.isDarwin [ Security ];
+
+  meta = with stdenv.lib; {
     description = "Commandline tool to display information about unicode characters";
     homepage = "https://github.com/antifuchs/chars";
     license = licenses.mit;
diff --git a/pkgs/top-level/all-packages.nix b/pkgs/top-level/all-packages.nix
index 7a00e1ec5bf4..6d58e2a81455 100644
--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -961,7 +961,9 @@ in
 
   charm = callPackage ../applications/misc/charm { };
 
-  chars = callPackage ../tools/text/chars { };
+  chars = callPackage ../tools/text/chars {
+    inherit (darwin.apple_sdk.frameworks) Security;
+  };
 
   ec2_api_tools = callPackage ../tools/virtualization/ec2-api-tools { };
 
```

Fixes building on darwin.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 103363` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>chars</li>
  </ul>
</details>

**Comment by bbigras** [comment]
Thanks @SuperSandro2000 !

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 103363` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>chars</li>
  </ul>
</details>

**Pull request merged**